includes/classes/InternalConnections/NetworkSiteConnection.php

```diff
@@ -201,28 +208,21 @@ public function push( $post, $args = array() ) {
 		}
 
 		/**
-		 * Allow bypassing of all media processing.
+		 * Allow bypassing of all meta processing.
 		 *
-		 * @hook dt_push_post_media
+		 * @hook dt_push_post_meta
 		 *
-		 * @param {bool}       true           If Distributor should push the post media.
+		 * @param {bool}       true           If Distributor should push the post meta.
 		 * @param {int}        $new_post_id   The newly created post ID.
-		 * @param {array}      $post_media    List of media items attached to the post, formatted by {@link \Distributor\Utils\prepare_media()}.
+		 * @param {array}      $post_meta     Meta attached to the post, formatted by {@link \Distributor\Utils\prepare_meta()}.
 		 * @param {int}        $post_id       The original post ID.
 		 * @param {array}      $args          The arguments passed into wp_insert_post.
 		 * @param {Connection} $this          The distributor connection being pushed to.
 		 *
-		 * @return {bool} If Distributor should push the post media.
+		 * @return {bool} If Distributor should push the post meta.
 		 */
-		if ( apply_filters( 'dt_push_post_media', true, $new_post_id, $post_media, $post_id, $args, $this ) ) {
-			Utils\set_media( $new_post_id, $post_media, [ 'use_filesystem' => true ] );
-		};
-
-		$media_errors = get_transient( 'dt_media_errors_' . $new_post_id );
-
-		if ( $media_errors ) {
-			$output['push-errors'] = $media_errors;
-			delete_transient( 'dt_media_errors_' . $new_post_id );
+		if ( apply_filters( 'dt_push_post_meta', true, $new_post_id, $post_meta, $post_id, $args, $this ) ) {
+			Utils\set_meta( $new_post_id, $post_meta );
```

Comment:
add_filter/remove_filter for the new method would be added either side of this line.